build_spec.yml

```diff
@@ -10,6 +10,7 @@ env:
   exportedVariables:
     - testEnv
     - minor_version
+    - oci_trigger_commit_hash
```

Comment:
if you are intending to export a predefined system variables then please use all caps one: OCI_TRIGGER_COMMIT_HASH

please test the change once by creating a build run and see in the build output section if this variable is exported as intended by you.